src/types/onyx/InvitedEmailsToAccountIDsDraft.ts

```diff
@@ -0,0 +1,3 @@
+type InvitedEmailsToAccountIDsDraft = Record<string, number>;
```

Comment:
You can use this type in `src/libs/actions/Policy.ts` too, there are a couple of places there that can use this I think.